l10n_es_facturae/data/data_res_country.xml

```diff
@@ -0,0 +1,742 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate='0'>
+        <record id="base.ad" model="res.country">
+            <field eval="&quot;AND&quot;" name="code_3166"/>
```

Comment:
Si pones `<field name="code_3166">AND</field>` te ahorras todos los `&quot;` que ocupan más bytes y lo hace menos legible. Salvemos unos cuántos árboles digitales! :stuck_out_tongue_winking_eye: